src/onnx/parse_pooling.cpp

```diff
@@ -159,6 +198,11 @@ struct parse_pooling : op_parser<parse_pooling>
 
         if(not slice_start.empty())
         {
+            if(in_shape.dynamic())
```

Comment:
Would it be able to handle if input is dynamic but fixed shape ?

Reply:
If all dimensions are fixed then we should probably make a static shape during construction.